modules/tutorials/partials/retrieve-documents-using-query-bench-section.adoc

```diff
@@ -0,0 +1,152 @@
+// suppress inspection "JsonStandardCompliance" for whole file
+// suppress inspection "SqlNoDataSourceInspection" for whole file
+// suppress inspection "SqlResolve" for whole file
+= Using the Query Editor
+
+Return to the admin console, and click on the menu:Query[] item on the left-hand menu.
+
+This will take you to the query workbench. The workbench has a few filter fields that'll make it much easier to narrow down our search criteria.
+
+image::set-query-filters.png[]
+
+Use the two dropdown items to select the `student-bucket` and the `art-school-scope`. This narrows the scope of the queries, meaning you don't have to add the name of the bucket and the scope to your queries.
+
+Okay, let's try a simple query to retrieve all the course in our collection.
+
+Type the following query into the query editor field:
+
+[source, couchbasequery]
```

Comment:
Should be n1ql, which is recognized by the hljs highlighter.
If we want couchbasequery as a synonym (do we?) then should add it with similar to https://github.com/couchbase/docs-ui/pull/119/commits/41af4b125e1faff6a47b1ef75bb06a690e384229